docs/07-AdvancedUsage.md

```diff
@@ -141,7 +136,27 @@ class MathTest extends \Codeception\TestCase\Test
 ?>
 ```
 
-It is usually preferable to make `_inject()` method `protected` or `private` for not to confuse it with `public` test methods. `_inject()` will be invoked just once right after creation of TestCase object (either Cest or Test).
+However, Dependency Injection is not limited for this. It allows you to **inject any class**, which can be constructed with arguments known to Codeception.
+
+In order to make auto-wiring work, you will need to implement `_inject()` method with the list of desired arguments. It is important to speicfy the type of arguments, so Codeception could guess which objects are expected to receive. `_inject()` will be invoked just once right after creation of TestCase object (either Cest or Test). Dependency Injection will also work in similar manner for Helper and Actor classes.
```

Comment:
so Codeception **~~could~~can** guess which objects are expected to **be** receive**d**.

after creation of **the** TestCase object

work in **a** similar manner for Helper